packages/shared/src/models/user.ts

```diff
@@ -1,8 +1,10 @@
 export type User = {
   id: number;
-  firstName: string;
-  lastName: string;
+  githubId: number;
+  firstName: string | null;
+  lastName: string | null;
   email: string;
+  image: string;
 };
 
 export type GetAllUsersResponse = User[];
```

Comment:
@pietruszka you're right. It was only created to demonstrate and test model sharing between server and client. It could be removed along with the entire related service, controller, and frontend handler. Saying so, I don't think it should be done in this PR as it would unnecessarily expand the scope of this task. I will create a separate issue for this.